### PR TITLE
A TLD-only domain fails to work with join-path

### PR DESCRIPTION
**Pull request by krainboltgreene**

### CODE REVIEW

**Comment by krainboltgreene** [comment]
Hey @michaelgraham, let me know when you get a chance to release this, but no rush.

**Comment by krainboltgreene** [comment]
screen shot 2017-01-10 at 11 19 48 am

File: src/sdk/request.js [hunk 1]
```diff
@@ -32,7 +32,10 @@ function request (spec = {}) {
 
       fetchConfig = {...{method: defaultMethod}, ...fetchConfig}
 
-      window.fetch(joinPath(origin, url), fetchConfig)
+      const path = replace(/^\//, '', `${url}`)
```
Comment (michaelgraham) [inline src/sdk/request.js]: `const path = replace(/^\//, '', url)`

File: src/sdk/request.js [hunk 2]
```diff
@@ -32,7 +32,10 @@ function request (spec = {}) {
...
+      const uri = replace(/\/$/, '', `${origin}`)
```
Comment (michaelgraham) [inline src/sdk/request.js]: `const uri = replace(/\/$/, '', origin)`

File: src/sdk/request.js [hunk 3]
```diff
@@ -32,7 +32,10 @@ function request (spec = {}) {
...
+
+      window.fetch(`${uri}/${path}`, fetchConfig)
```
Comment (michaelgraham) [inline src/sdk/request.js]: what would you think of `[uri, path].join('/')` instead?

**Pull request merged**